tests/Aggregate/ConfigurableAggregateTranslatorTest.php

```diff
@@ -0,0 +1,287 @@
+<?php
+/*
+ * This file is part of the prooph/event-store.
+ * (c) 2014-2015 prooph software GmbH <contact@prooph.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * Date: 8/22/15 - 9:42 PM
```

Comment:
Date format in copyright should be consistent, you use always different formats ;)